CHANGES.rst

```diff
@@ -1,224 +0,0 @@
-Changes
```

Comment:
I chose to outright delete `CHANGES.rst` for simplicity... this is a bit heavy-handed, I also considered simply renaming it to `CHANGES_ARCHIVED.rst`, but decided against it for three reasons:
1. We've been maintaining changes in the GitHub releases UI for a while, so there's already a recent changelog that goes back as far as anyone who's trying to actively use this project would really need.
2. Some tooling (Dependabot, RenovateBot, etc) may look for `*change*` filenames when auto-populating changelogs... so keep life simple and force them to look for the changelog URL from `setup.cfg` and not let it accidentally pickup a `CHANGES_ARCHIVED` filename.
3. If we leave the file around, then at some point in the future we'll eventually delete it... life is simpler if we just nuke it now. It's still in git history for anyone who really needs it.

Open to discussion if other folks feel strongly that we should keep it.

Reply:
Hi,

I think your reasoning for dropping the file is sound. The only comment I have is around "hidden expectations". As an end user I usually dart directly for a changelog file in the repo to get a glance of what's either released or coming soon. If I don't see that file, it's an extra step to check the releases. Some folks may not know it exists there. I don't think this is an end of the world scenario though and if it makes maintaining the project easier, it's a win.

From a technical standpoint the only downside is it couples things to GitHub directly but I think in the grand scheme of things this is a non-issue?

Reply:
You both make good points. I don't feel strongly either way, and think that deleting the file now is probably best. I don't think it will be too difficult to see changes with the new method, and people will find what they need.